src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.c

```diff
@@ -358,7 +357,7 @@ static grpc_ares_request *grpc_dns_lookup_ares_impl(
     grpc_ares_request_ref(r);
     char *service_name;
     gpr_asprintf(&service_name, "_grpclb._tcp.%s", host);
-    ares_query(*channel, service_name, ns_c_in, ns_t_srv, on_srv_query_done_cb,
+    ares_query(*channel, service_name, 1 /*ns_c_in*/, 33 /*ns_t_srv*/, on_srv_query_done_cb,
```

Comment:
good question, can you ask this question in https://github.com/c-ares/c-ares/issues/131 please?

To me it looks like the `ares_query` is just an API that can be used with arbitary ARPA codes that might be standardized somewhere. Doe to that, they are *not* c-ares constants but more general ARPA constants.